Adding 6-info-page template. closes #2384

webinista opened the pull request:
- Renaming property from Template to template as needed.
- Changing some template values from 6-info-page to 1-topic-landing

Blocks #2366

webinista committed:
Adding 6-info-page template. closes #2384

- Renaming property from Template to template as needed.
- Changing some template values from 6-info-page to 1-topic-landing

webinista committed:
Merge branch 'master' into issue#2384

Commit:
Merge branch 'master' into issue#2384

emilyville commented on _disability-benefits/claims-process/claim-types/predischarge-claim/tools.md:
```diff
@@ -1,29 +1,6 @@
-<div class="row" markdown="0">
-<div class="small-12 columns" markdown="1">
-
 Coming soon.
```
Should we remove this page?

webinista replied:
I think so, yes. I erred on the side of leaving it around in case it's linked to from somewhere. But we can remove the links too.

emilyville commented on _disability-benefits/conditions/ptsd.md:
```diff
@@ -51,17 +35,13 @@ Yes, if:
 
 You
 
```
the blue box needs to end here!

emilyville commented on _disability-benefits/conditions/special-claims/dentistry.md:
```diff
@@ -2,29 +2,6 @@
-<div class="primary" markdown="0">
-<div class="row" markdown="0">
-<div class="small-12 columns usa-content" markdown="1">
+template: 1-topic-landing
```
this page looks broken - the yaml is showing

emilyville commented on _education/gi-bill/post-9-11.md:
```diff
@@ -38,15 +24,13 @@ You and your dependents
 - A books and supplies stipend (up to $1,000 per academic year)
 
-<hr>
+-------
```
for me a bunch of divs are printing at the bottom of this page

emilyville commented on _education/gi-bill/post-9-11/bill-transfer.md:
```diff
@@ -1,28 +1,6 @@
-<div class="row" markdown="0">
-<div class="small-12 columns" markdown="1">
-
 Coming soon.
```
should we remove this page?

emilyville commented on _education/tools-programs.md:
```diff
@@ -1,7 +1,7 @@
+template: 1-topic-landing
 relatedlinks:
   - url: /education/tools-programs/education-career-counseling/
     title: Education and Career Counseling
```
there is a double apply button on this page

webinista committed:
Fixing markup on some pages. Removing "Coming soon" pages.

deleted:    _disability-benefits/claims-process/claim-types/predischarge-claim/tools.md
modified:   _disability-benefits/conditions/ptsd.md
modified:   _disability-benefits/conditions/special-claims/dentistry.md
modified:   _education/gi-bill/post-9-11.md
deleted:    _education/gi-bill/post-9-11/bill-transfer.md
modified:   _education/tools-programs.md

emilyville commented:
Looks like you got most of the comments but there is a couple more - sending back your way! 

Big and good change!!

webinista committed:
Merge branch 'master' into issue#2384

webinista committed:
Merge branch 'master' into issue#2384

emilyville commented:
Looks good!!
